packages/block-editor/README.md

```diff
@@ -122,6 +122,10 @@ Undocumented declaration.
 
 Undocumented declaration.
 
+<a name="BlockPreview" href="#BlockPreview">#</a> **BlockPreview**
+
+Undocumented declaration.
```

Comment:
Can we add a comment on the component to avoid this?

Reply:
Done